conda

```diff
@@ -13,10 +13,28 @@ conda create --name 'ENVNAME' python=3.10
 # To install from requirements.txt
 conda install --file requirements.txt
 
+# To clone an environment
+conda create --clone 'ENVNAME' -n 'NEWENV'
```

Comment:
```suggestion
conda create --clone "<old_environment_name>" --name "<new_environment_name>"
```

- It's better to use long options because they easily memorable for users.
- We can use more descriptive placeholder names here.